src/app/shell/personal-cabinet/parent/children/children.component.ts

```diff
@@ -8,30 +8,31 @@ import { ConfirmationModalWindowComponent } from '../../../../shared/components/
 import { PaginationConstants, Constants, ModeConstants } from '../../../../shared/constants/constants';
 import { ModalConfirmationType } from '../../../../shared/enum/modal-confirmation';
 import { NavBarName } from '../../../../shared/enum/navigation-bar';
-import { Child } from '../../../../shared/models/child.model';
+import { Child, ChildrenParameters } from '../../../../shared/models/child.model';
 import { PaginationElement } from '../../../../shared/models/paginationElement.model';
 import { PushNavPath } from '../../../../shared/store/navigation.actions';
-import { SetFirstPage, SetChildrensPerPage, OnPageChangeChildrens } from '../../../../shared/store/paginator.actions';
-import { PaginatorState } from '../../../../shared/store/paginator.state';
 import { GetUsersChildren, DeleteChildById } from '../../../../shared/store/parent.actions';
 import { ParentState } from './../../../../shared/store/parent.state.';
 import { SearchResponse } from '../../../../shared/models/search.model';
+import { Util } from '../../../../shared/utils/utils';
 
 @Component({
   selector: 'app-children',
   templateUrl: './children.component.html',
-  styleUrls: ['./children.component.scss'],
+  styleUrls: ['./children.component.scss']
 })
 export class ChildrenComponent extends ParentComponent implements OnInit, OnDestroy {
   readonly ModeConstants = ModeConstants;
 
-  @Select(PaginatorState.childrensPerPage)
-  childrensPerPage$: Observable<number>;
   @Select(ParentState.children)
   childrenCards$: Observable<SearchResponse<Child[]>>;
   childrenCards: SearchResponse<Child[]>;
 
   currentPage: PaginationElement = PaginationConstants.firstPage;
+  childrenParameters: ChildrenParameters = {
+    searchString: '',
+    tabTitle: undefined
```

Comment:
I see you have to approaches, somewhere you assign null and in other cases - undefined. Keep one approach please